### ISSUE TITLE
What is the use of Outlet from 'react-router-dom' ? 

### ISSUE THREAD

**Issue by utkarsh-1602** [utkarsh-1602/NebulaNetwork-social-media-app#4]
my `RootLayout.tsx`:
```

import Bottombar from "@/components/ui/shared/Bottombar"
import LeftSidebar from "@/components/ui/shared/LeftSidebar"
import TopBar from "@/components/ui/shared/TopBar"
import { Outlet } from "react-router-dom"

const RootLayout = () => {
    return (
        <div className="w-full md:flex">
            <TopBar />
            <LeftSidebar />

            <section className="flex flex-1 h-full">
                <Outlet />
            </section>

            <Bottombar />

        </div>
    )
}

export default RootLayout
```

Explain the use of Outlet

**Comment by utkarsh-1602** [comment]
In React Router, the Outlet component is used to render the child routes within a parent route. It serves as a placeholder where the child routes defined in your route configuration will be rendered. This is particularly useful when you have nested routes.

Under the Same directory of `RootLayout.tsx`, the pages folder consists of Home, Explore, LikedPosts.tsx Components. Those Components are Child routes. 

In summary, the Outlet component is used to render the child routes within a layout or parent route, allowing you to create complex nested layouts in your React application.
